l2geth/core/tx_pool.go

```diff
@@ -560,33 +553,21 @@ func (pool *TxPool) validateTx(tx *types.Transaction, local bool) error {
 		return ErrUnderpriced
 	}
 	// Ensure the transaction adheres to nonce ordering
```

Comment:
This block of code should actually still stay, otherwise it will introduce a vulnerability. This is because we send transactions directly to the miner, they do not wait in the mempool